enos/enos.py

```diff
@@ -24,7 +24,7 @@
   ssh-tunnel     Print configuration for port forwarding with horizon.
   tc             Enforce network constraints
   info           Show information of the actual deployment.
-  destroy        Destroy the resources used by the deployment
+  destroy        Destroy the deployment and optionnaly the related ressurces.
```

Comment:
typos: `optionally` and `related resources`
Note that `ressource` is found multiple times in the rest of the commit